rest_api/aggregation/database_side/stages.py

```diff
@@ -115,4 +115,12 @@ def merge_analysis_driver_procs(id_field, projection=None):
     for k in projection:
         group[k] = {'$first': '$' + k}
     stages.append({'$group': group})
+    stages.append(
+        lookup(
+            'analysis_driver_stages',
+            'most_recent_proc.proc_id',
+            'analysis_driver_proc',
+            'most_recent_proc.stages'
+        )
+    )
```

Comment:
Is this really necessary? 
Are the analysis_driver_stages used anywhere after pulling a sample or a run.
The downside of adding this here is we have another lookup done for every query to runs or samples which will slow them all down even if the stages are not used.